### ISSUE TITLE
Feature Request - Device Memory Slots pane - add custom note field

### ISSUE THREAD

**Issue by Scachi** [J2Kbr/GtunerIV#216]
For some games I have the same script installed to multiple slots with different settings.
The Slot pane looks something like:
````
1: Division v1.20
   Scachi (date time)
2: Division v1.20
   Scachi (date time)
3: Division v1.20
   Scachi (date time)
````
A few days later and I got lost which slot I used for the specific weapon/gear sets.

It would help a lot to have an additional line for some custom text/note like this:
````
1: Division v1.20
   Scachi (date time)
   Gear Set: Striker
2: Division v1.20
   Scachi (date time)
   Gear Set: Nomad
3: Division v1.20
   Scachi (date time)
   PVP Loadout
````
The third lines text can be as small as the second line already is, or the custom text may only show up as a hint on move hovering over the slots info text lines to save some space.
or if a note is set then hide the date/time and show the note instead like this (I like this one the most):
````
1: Division v1.20
   Scachi (Gear Set: Striker)
2: Division v1.20
   Scachi (Gear Set: Nomad)
3: Division v1.20
   Scachi (PVP Loadout)
````

Setting the custom text may be possible to do with right click on a slot and add a menu entry like "Add slot note" or something like that.
Updating/installing a script in a slot should also keep the note when holding down the ctrl key.

**Comment by J2Kbr** [comment]
Feature implemented on Gtuner 1.00RC-85 (as suggestion no. 3: via context menu, replacing the date/time, hold ctrl to keep annotation).

**Comment by Scachi** [comment]
This is really cool. Thank you very much !

![grafik](https://user-images.githubusercontent.com/22262242/50735890-de6dc780-11b5-11e9-8ea6-233aeb514276.png)
